### PR TITLE
add ADS toggle option + fix scoping prediction

### PR DESCRIPTION
**Pull request by 2048khz-gachi-rmx**
!!! UNTESTED IN SP !!!
!!! PLEASE TEST IN SP BEFORE MERGING !!!

see: https://trello.com/c/xAesYmil/4-toggle-aim

https://user-images.githubusercontent.com/44749188/109384628-c7a00700-7920-11eb-9ce0-48d00e011909.mp4

### CODE REVIEW

**Comment by TheOnly8Z** [comment]
Does not work in SP or listen servers without ``IsFirstTimePredicted()``. I have edited the code to include the check.

**Comment by TheOnly8Z** [comment]
@2048khz-gachi-rmx Please check if the edited code works in your testing environment (I presume a dedi server?), otherwise LGTM.

**Comment by 2048khz-gachi-rmx** [comment]
it will cause prediction issues if you don't do this in predicted hook calls; i'll try checking if game.SinglePlayer() checks would work

**Comment by TheOnly8Z** [comment]
Not having IsFirstTimePredicted() will also cause issues in listen servers (as the host, at least).

**Comment by 2048khz-gachi-rmx** [comment]
check now; seems to be working in all realms

File: lua/arccw/client/cl_convars.lua [hunk 1]
```diff
@@ -105,7 +105,7 @@ ArcCW.ClientConVars = {
     ["arccw_vm_sway_speedmult"]        = { def =  1 },
     ["arccw_vm_sway_rotatemult"]       = { def =  1 },
 
-    -- ["arccw_toggleads"]               = { def = 0, usri = true },
+    ["arccw_toggleads"]               = { def = 0, usri = true },
```
Comment (2048khz-gachi-rmx) [inline lua/arccw/client/cl_convars.lua:108]: trailing space :DDDDDDDDDDDD

**Comment by TheOnly8Z** [comment]
LGTM.

**Pull request merged**